packages/frontend/src/components/map/Map.tsx

```diff
@@ -0,0 +1,39 @@
+'use client';
+import React, { useEffect } from 'react';
+import { MapContainer, Marker, TileLayer, Popup } from 'react-leaflet';
+// import L from 'leaflet';
```

Comment:
**nitpick (non-blocking):** Remove commented part of code from PR.  This should have been caught during linting but I guess our linting is not strict